docs/source-app/workflows/byoc/index.rst

```diff
@@ -5,62 +5,287 @@ Run Apps on your own cloud (BYOC)
+----
+
+*************************
+Step 2: Install Lightning
+*************************
+
+Lightning must be installed before you can create a Lightning BYOC cluster.
+
+----
+
+Who should install Lightning?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once your AWS account is ready, IT’s job is done. That means you are the one who will be doing this (researchers, data scientists, Lightning App creators).
+
+----
+
+Install Lightning
+^^^^^^^^^^^^^^^^^
+
+From your local machine use the following command to install Lightning:
+
+.. code:: bash
+
+    python -m pip install lightning
+
+We have versioned pinned requirements at the time of the writing. Feel free to use the newer versions:
+
+.. code:: bash
+
+    torch==1.1
+    torchvision
+    pytorch_lightning
+
+----
+
+******************************************
+Step 3: Set up your Lightning BYOC cluster
+******************************************
+
+Creating a cluster is a one time operation, unless you intend to add more clusters.
+
+----
 
-After submitting a cluster creation request, the Lightning Control Plane creates the required cloud infrastructure on the user account. This
-sets up a new Lightning Cluster along with a Lightning Kubernetes Operator.
+Who should set up your Lightning BYOC cluster?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+With Lightning installed, researchers, data scientists, Lightning App creators, and more, can now start creating and monitoring clusters.
+
+----
 
-*******************************
 Create a Lightning BYOC cluster
-*******************************
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 You must have your cloud configured before you try and create a BYOC cluster.
 
-And to make your life a little easier, we've made a `Terraform module to help with that <https://github.com/Lightning-AI/terraform-aws-lightning-byoc>`_.
-
-Create a Lightning BYOC cluster using the following command:
+Create a Lightning BYOC cluster using the following command from your local machine:
 
 .. code:: bash
 
-   lightning create cluster <cluster-name> <cloud-provider-parameters>
+     python -m lightning create cluster <cluster-name> --provider aws --role-arn <ARN-you-copied-earlier> --external-id <external-id-you-copied-earlier> --region <aws-region> <other-cloud-provider-parameters> --wait
+
+.. note::
+
+    Most AWS regions should work, but we recommend using one of the following AWS regions when creating a cluster:
+
+    * us-east-1
+    * us-west-2
+    * eu-west-1
+    * ap-south-1
 
 Here's an example:
 
 .. code:: bash
 
-   lightning create cluster my-byoc-cluster --provider aws --role-arn arn:aws:iam::1234567890:role/lai-byoc --external-id dummy --region us-west-2 --instance-types t3.xlarge --enable-performance
+    python -m lightning create cluster my-byoc-cluster --provider aws --role-arn arn:aws:iam::1234567890:role/lai-byoc --external-id dummy --region us-west-2 --instance-types t3.xlarge –-wait
```

Comment:
here too

Reply:
@awaelchli 

The dev team tells me this is required. @nmiculinic please verify.

Reply:
Does that mean I'm not part of the dev team? 😢 
If this is really required, I think we have a problem and we need to fix it asap. I am pretty confident that this is not the UX we are going for. @edenlightning

Reply:
@awaelchli 

This feature is still early access. That doesn't forgive everything, but maybe a few allowances can be made?

Reply:
@awaelchli 

Also of course you're part of the dev team. Big love to you.

Nash, Neven, and Raphael we're simply the ones that provided feedback on the initial content.

Reply:
If your env is healthy, you do not need it, but there can easily happen that our default `pip` may point to no default `python`, so to use this notation is a safer way...
in particular, if your OS is mixing pip and Conda it could be quite hard to get things right...